docs/migrate/rockset/index.md

```diff
@@ -0,0 +1,355 @@
+products seamlessly, and supply your engineers and continuous integration
+systems with dedicated instances of CrateDB, instead of needing to run
+them against a cloud-only service, which is slowing down development,
+mostly due to API rate-limiting measures.
+::::
+
+:::::
+
+
+## Learn
+Learn how to migrate your database workloads from Rockset to CrateDB.
+
+:::::{grid} 1 1 2 2
+:gutter: 3
+
+::::{grid-item-card}
+:::{rubric} SQL Query Language
+:::
+CrateDB's lingua franca is SQL, ready for big data, very similar to
+Rockset's SQL dialect.
+- [CrateDB SQL]
+- [Advanced Querying]
+:::{rubric} Migrating queries from Rockset to CrateDB
+:::
+Because both Rockset and CrateDB use SQL, there is no need for your teams to
+learn a new query language. There are a few differences in the SQL dialect,
+where we provide relevant support information to make transitioning easier.
+:::{toctree}
+Migrate Queries <query>
+:::
+::::
+
+::::{grid-item-card}
+:::{rubric} Integrating data from other sources into CrateDB
+:::
+With Rockset, as a complete data warehouse system, integrating data from other
+sources is a nobrainer, and provides exceptional UX/DX.
+
+CrateDB offers a wide range of integration capabilities, is compatible
+with the PostgreSQL wire protocol, and offers adapter components to
+integrate with applications and frameworks.
+
+- [Ecosystem Catalog]
+- [Integration Tutorials I]
+- [Integration Tutorials II]
+- [Software Development Kit]
+
+:::{rubric} Integrating CrateDB technologies into Rockset-based infrastructure
+:::
+The [Rockset HTTP API Adapter for CrateDB] is an experiment to provide
+CrateDB's features through an API that is compatible with the Rockset HTTP
+API, so client programs and libraries can work unmodified.
+
+It has been verified to work for the most basic API calls with plain
+HTTP requests using curl or HTTPie, the Rockset CLI, and Java, JavaScript,
+and Python example programs.
+::::
+
+:::::
+
+
+## What's Next
+
+Our engineers are working on software components that yield immediate leverage
+when aiming to migrate data integration pipelines, based on popular requests
+from Rockset customers.
+
+:::::{grid} 1 1 2 2
+:gutter: 2
+:padding: 0
+
+::::{grid-item-card}
+[Zero-ETL] [DX]/[UX], to make data ingestion work on your fingertips.
+
+- [Amazon DynamoDB Streams]
+- [Amazon Kinesis Data Streams]
+- [Apache/Confluent Kafka Streams]
+- [MongoDB Atlas Change Streams]
+
+:::{todo}
+Link to alpha/beta/wip variants of corresponding adapters.
+:::
+::::
+
+::::{grid-item-card}
+Our teams are working on improving SQL support and integrating relevant
+data integration features into our managed solutions on [CrateDB Cloud].
+
+Feel free to reach out to us any time, in order to share your use cases
+and workload insights. We will be happy to take them into consideration.
+
+:::{todo}
+Please enumerate a few items which will be served next, and/or suggest a few
+more words to fill the void right here.
+:::
+::::
+
+:::::
+
+
```

Comment:
Here, at the bottom of the page, I would like to add embeds for one or two canonical videos which absolutely make CrateDB shine, not necessarily from our own pen. Do you have any in mind which would be applicable in this context?

Reply:
Diverted to GH-104.